charmcraft/metafiles/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For further info, check https://github.com/canonical/charmcraft
+
+"""Submodule for handling the all extra yaml files."""
+
+__all__ = ["manifest", "metadata"]
```

Comment:
```suggestion
from charmcraft.metafiles import manifest, metadata

__all__ = ["manifest", "metadata"]
```

Upcoming linter changes are probably going to include complaints about exports that aren't explicitly imported :-)